recipes/unicorn-binance-local-depth-cache/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "unicorn-binance-local-depth-cache" %}
+{% set version = "0.7.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/unicorn-binance-local-depth-cache-{{ version }}.tar.gz
+  sha256: 2ee1e2c199ae6310816af8aaf7c9f1eda2d2120446462eacb4f9dc7a371feb43
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - requests
+    - unicorn-binance-rest-api
+    - unicorn-binance-websocket-api >=1.40.5
+  run:
+    - python >=3.7
+    - requests
+    - unicorn-binance-rest-api
+    - unicorn-binance-websocket-api >=1.40.5
+
+test:
+  imports:
+    - unicorn_binance_local_depth_cache
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://www.lucit.tech/unicorn-binance-local-depth-cache.html
+  summary: 'A local Binance DepthCache Manager for Python that supports multiple depth caches in one instance in a easy, fast, flexible, robust and fully-featured way.'
+  description: |
+    The Python package UNICORN Binance Local Depth Cache provides a local depth_cache for the Binance Exchanges Binance (+Testnet), Binance Futures 
+    (+Testnet).
```

Try to wrap this to be shorter:
```suggestion
    The Python package UNICORN Binance Local Depth Cache provides a local
    depth_cache for the Binance Exchanges Binance (+Testnet), Binance Futures
    (+Testnet).
```